package.json

```
@@ -14,7 +14,10 @@
     "@astrojs/check": "^0.3.4",
     "@astrojs/node": "^7.0.3",
     "@astrojs/tailwind": "^5.1.0",
+    "@auth/core": "^0.18.6",
+    "@auth/drizzle-adapter": "^0.3.12",
```

```suggestion
    "@auth/drizzle-adapter": "^0.5.0",
```
I had to update this adapter